cmd/search/http.go

```diff
@@ -117,7 +119,7 @@ func (o *options) handleIndex(w http.ResponseWriter, req *http.Request) {
 	}
 
 	var searchTypeOptions []string
-	for _, searchType := range []string{"bug+junit", "bug", "junit", "build-log", "all"} {
+	for _, searchType := range []string{"bug+junit", "issue", "bug", "bug+issue", "junit", "build-log", "all"} {
```

Comment:
Is it possible to do bug+issue+junit as the default? If someone is searching for a problem w/ a test, it could help to have jira issues shown as well as bugs.

Reply:
Done; bug+issue+junit is added to the options list and made the default selection.